mobile/src/people/employee-card-with-avatar.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react';
+import { Animated, Easing, View, Image, Dimensions, StyleSheet, FlatList, ListRenderItemInfo } from 'react-native';
+import { Avatar } from './avatar';
+import { Employee } from '../reducers/organization/employee.model';
+import { StyledText } from '../override/styled-text';
+import { companyItemStyles as styles, companyTinyItemStyles as tinyStyles} from './styles';
+
+interface EmployeeCardWithAvatarProps {
+    employee?: Employee;
+    departmentAbbreviation?: string;
+    chiefId?: string;
+    employees?: Employee[];
+    leftNeighbor?: Employee;
+    rightNeighbor?: Employee;
+    treeLevel?: number;
+    stretchToFitScreen?: boolean;
+}
+
+export class EmployeeCardWithAvatar extends Component<EmployeeCardWithAvatarProps> {
+    public state = {
+        fadeInAnim: new Animated.Value(0),
+        fadeOutAnim: new Animated.Value(1),
+        isNeighboursAvatarsVisible: true
+    };
+
+    public revealNeighboursAvatars(visibility: boolean) {
+        this.setState({isNeighboursAvatarsVisible: visibility});
+    }
+
+    public componentDidMount() {
+        // Animated.timing(
+        //   this.state.fadeInAnim,
+        //   {
+        //     toValue: 1,
+        //     duration: 3000,
+        //   }
+        // ).start();
+
+        // Animated.timing(
+        //     this.state.fadeOutAnim,
+        //     {
+        //       toValue: 0,
+        //       duration: 2000,
+        //     }
+        // ).start();
+    }
+
+    public render() {
+        const employee = this.props.employee;
+        const neighboursAvatarsVisibility = this.state.isNeighboursAvatarsVisible;
+        const { fadeInAnim, fadeOutAnim } = this.state;
+        const photo = employee ? employee.photo : null;
+
+        Animated.timing(
+            this.state.fadeInAnim,
+            {
+              toValue: 1,
+              duration: 1000,
+            }
+          ).start();
```

Comment:
render MUST be a pure function. In other words, it's not supposed to have this call here. what you should do is to take a value from fadeInAnim here.

Reply:
addressed